### DISCUSSION TITLE
dll dependencies on windows (gnu toolchain)

### DISCUSSION THREAD

**Discussion by aaronps** [fltk-rs/fltk-rs#1176] [General]
Using `fltk-bundled` I was getting different results on two different computers: a simple application would require a bunch of msys2 dll's to run when build in one computer, but didn't require any dll in the other.

After further investigation I found that in the one that requires the extra dll's I had installed (maybe as a dependency long ago) **libgdiplus** which was linked first and added the extra dll files as dependencies, 28 files in total. After uninstalling it with `pacman -R ming-w64-x86_64-libgdiplus` the problem went away.

Maybe you could add this info to the FAQ or something.

**Comment by MoAlyousef** [comment]
Thanks for the info. I’ll add a note in the faq.
